packages/react-components/react-alert/src/stories/Alert/AlertAppearance.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { Alert } from '@fluentui/react-alert';
+import { Title3 } from '@fluentui/react-text';
+
+const renderAlerts = (appearance: 'primary' | 'inverted') => (
+  <div style={{ flexGrow: 1 }}>
+    <Title3>{appearance}</Title3>
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '10px' }}>
+      <Alert
+        appearance={appearance}
+        intent="success"
+        action={{
+          children: 'Undo',
+        }}
```

Comment:
```suggestion
        action='Undo'
```

nit: You can use shorthand form for this slot as other props are not used